system_tests/espresso-e2e/.env

```diff
@@ -12,32 +12,48 @@ L1_BLOCK_TIME_SEC=3
 ASYNC_STD_THREAD_COUNT=1
 
 # Internal port inside container
-ESPRESSO_WEB_SERVER_PORT=40000
+ESPRESSO_CDN_SERVER_PORT=40000
+ESPRESSO_CDN_SERVER_METRICS_PORT=9090
 ESPRESSO_ORCHESTRATOR_PORT=40001
-ESPRESSO_CONSENSUS_SERVER_PORT=40002
-ESPRESSO_DA_SERVER_PORT=40003
-ESPRESSO_SEQUENCER_DA_SERVER_URL=http://da-server:$ESPRESSO_WEB_SERVER_PORT
-ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL=http://consensus-server:$ESPRESSO_WEB_SERVER_PORT
-ESPRESSO_SEQUENCER_ORCHESTRATOR_URL=http://orchestrator:$ESPRESSO_ORCHESTRATOR_PORT
+ESPRESSO_ORCHESTRATOR_NUM_NODES=2
+ESPRESSO_ORCHESTRATOR_START_DELAY=5s
+ESPRESSO_ORCHESTRATOR_NEXT_VIEW_TIMEOUT=30s
+ESPRESSO_ORCHESTRATOR_BUILDER_TIMEOUT=2s
+ESPRESSO_SEQUENCER_CDN_ENDPOINT=marshal-0:${ESPRESSO_CDN_SERVER_PORT}
+ESPRESSO_SEQUENCER_ORCHESTRATOR_URL=http://orchestrator:${ESPRESSO_ORCHESTRATOR_PORT}
 ESPRESSO_SEQUENCER_API_PORT=50000
+ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_PORT=42000
+ESPRESSO_SEQUENCER_MAX_BLOCK_SIZE=10kb
```

Comment:
Am I missing something here or do we need to pass in size as u64? Does clap do some magic interpretation?

Reply:
Do you get errors? It should be handled here: https://github.com/EspressoSystems/espresso-sequencer/blob/54487da20078051b7bedbf27ce25698d2f79fa46/sequencer/src/options.rs#L237

Reply:
never mind, just found our `parse_size` fn in sequencer.